dev-cpp/string-theory/string-theory-3.3.ebuild

```diff
@@ -3,13 +3,11 @@
 
 EAPI=7
 
-inherit cmake ${GIT_ECLASS}
```

Comment:
This should be in the string-theory commit, not in the ja2-stracciatella commit

Reply:
I dont know how to revert that - it got too complex for me to handle 4 commits for one game and it would break all the other commits i guess?

Reply:
Try `git checkout` to string-theory commit, create a new branch from it, amend the commit with this change, then rebase the ja2-stracciatella branch onto that new branch.

When I'm working on several dependent commits like that, I also often create several small commits with fixes, then use `git rebase -i` to apply those fixes to correct commits. That's not very nice, but works for me more or less. Actually, I'm missing some functionality from mercurial where it's easier to manage

Reply:
done

Reply:
I still see 58f3e859bcdfc98ee55a73014a4081ccbb306273 contains changes to string-theory and libsmacker